appveyor.yml

```diff
@@ -0,0 +1,17 @@
+build: off
```

Comment:
I'd be happy to back this out, but AppVeyor is a handle Windows CI service that is free for open source project.